src/llvm-alloc-opt.cpp

```diff
@@ -752,10 +752,11 @@ void Optimizer::moveToStack(CallInst *orig_inst, size_t sz, bool has_ref, AllocF
                 call->eraseFromParent();
                 return;
             }
-            if (pass.gc_loaded_func == callee) {
-                user->replaceUsesOfWith(orig_i, Constant::getNullValue(orig_i->getType()));
-                return;
-            }
+            //if (pass.gc_loaded_func == callee) {
```

Comment:
This deleted some code and then added some new code as a comment without any explanation. Either delete or add something describing why it is there and commented out maybe?

Reply:
I believe that you are looking only at the single commit.  It makes more sense if you look at the files changed (this PR now makes no changes to alloc-opt).

Reply:
Ah, okay.